File: src/main/java/io/github/mg138/tsbook/items/data/stat/set/ModifierType.java
package io.github.mg138.tsbook.items.data.stat.set;

import io.github.mg138.tsbook.items.data.stat.StatType;

import java.util.EnumSet;

public class ModifierType {
    public static final EnumSet<StatType> MODIFIER = EnumSet.of(
            StatType.MODIFIER_HELL,
            StatType.MODIFIER_MOBS,
            StatType.MODIFIER_PLAYER,
            StatType.MODIFIER_ARTHROPOD,
            StatType.MODIFIER_UNDEAD,
            StatType.MODIFIER_UNDERWATER
    );
}

File: src/main/java/io/github/mg138/tsbook/items/data/stat/util/set/ModifierType.kt
package io.github.mg138.tsbook.items.data.stat.util.set

import java.util.EnumSet
import io.github.mg138.tsbook.items.data.stat.StatType

object ModifierType {
    val types: EnumSet<StatType> = EnumSet.of(
        StatType.MODIFIER_HELL,
        StatType.MODIFIER_MOBS,
        StatType.MODIFIER_PLAYER,
        StatType.MODIFIER_ARTHROPOD,
        StatType.MODIFIER_UNDEAD,
        StatType.MODIFIER_UNDERWATER
    )
}